Jenkinsfile

```diff
@@ -33,8 +33,8 @@ timestampedNode('java') {
     timeout(time: 180, unit: 'MINUTES') {
         // See below for what this method does - we're passing an arbitrary environment
         // variable to it so that JAVA_OPTS and MAVEN_OPTS are set correctly.
-        withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m",
-                      "MAVEN_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m"]) {
+        withMavenEnv(["JAVA_OPTS=-Xmx4g -Xms512m -XX:MaxPermSize=1024m",
```

Comment:
this is also the wrong way - Maven should not need this amount of memory - if it does something is broken.

If tests are failing then configure surefire!